aws-lambda/go/README.md

```diff
@@ -0,0 +1,52 @@
+# Steps to test Sentry SDK integration with Node Lambda function:
+
+[Demo Video](https://www.loom.com/share/e3625614df2f44f3bb96286ad7212f99)
+
+## Create a development package
+1. Clone the git repo on your development machine.
+
+2. Build your executable by running
+
+```
+GOOS=linux go build main.go
```

Comment:
I suggest adding also the architecture, since soon more devs might be on ARM on new macs.

```suggestion
GOOS=linux GOARCH=amd64 go build main.go
```

Reply:
thanks, updated.